tests/dike_reinforcements/reinforcement_profile/berm_calculator/conftest.py

```diff
@@ -0,0 +1,88 @@
+from typing import Iterator
+
+import pytest
+
+from koswat.configuration.settings.koswat_scenario import KoswatScenario
+from koswat.configuration.settings.reinforcements.koswat_cofferdam_settings import (
+    KoswatCofferdamSettings,
+)
+from koswat.configuration.settings.reinforcements.koswat_piping_wall_settings import (
+    KoswatPipingWallSettings,
+)
+from koswat.configuration.settings.reinforcements.koswat_reinforcement_settings import (
+    KoswatReinforcementSettings,
+)
+from koswat.configuration.settings.reinforcements.koswat_soil_settings import (
+    KoswatSoilSettings,
+)
+from koswat.configuration.settings.reinforcements.koswat_stability_wall_settings import (
+    KoswatStabilityWallSettings,
+)
+from koswat.configuration.settings.reinforcements.koswat_vps_settings import (
+    KoswatVPSSettings,
+)
+from koswat.dike.profile.koswat_input_profile_base import KoswatInputProfileBase
+from koswat.dike_reinforcements.reinforcement_profile.berm_calculator.piping_berm_calculator import (
+    PipingBermCalculator,
+)
+
+
+@pytest.fixture(name="valid_scenario")
+def _get_valid_scenario_fixture() -> Iterator[KoswatScenario]:
+    yield KoswatScenario(
+        scenario_name="Test Scenario",
+        scenario_section="Test Section",
+        d_h=1.5,
+        d_s=2.0,
+        d_p=3.0,
+        crest_width=4.0,
+        waterside_slope=0.3,
+    )
+
+
+@pytest.fixture(name="valid_reinforcement_settings")
+def _get_valid_reinforcement_settings_fixture() -> Iterator[
+    KoswatReinforcementSettings
+]:
+    yield KoswatReinforcementSettings(
+        soil_settings=KoswatSoilSettings(),
+        vps_settings=KoswatVPSSettings(polderside_berm_width_vps=10),
+        piping_wall_settings=KoswatPipingWallSettings(
+            transition_cbwall_sheetpile=10.0,
+        ),
+        stability_wall_settings=KoswatStabilityWallSettings(
+            transition_sheetpile_diaphragm_wall=15.0,
+        ),
+        cofferdam_settings=KoswatCofferdamSettings(),
+    )
+
+
+@pytest.fixture
+def valid_input_profile() -> Iterator[KoswatInputProfileBase]:
```

Comment:
Follow the already used approach below by declaring the name and having a `_get` signature, I wold also use something more explicit as you can eventually end up overriding fixtures.
```suggestion
@pytest.fixture(name="valid_berm_calculator_input_profile")
def _get_valid_berm_calculator_input_profile_fixture() -> Iterator[KoswatInputProfileBase]:
```

Reply:
Adjusted, although the references become obfuscated by this.